neutrino-custom/lint.js

```diff
@@ -95,14 +95,15 @@ module.exports = neutrino => {
                     'no-void': 'error',
                     'no-with': 'error',
                     'object-curly-spacing': ['error', 'always'],
+                    'react/jsx-uses-react': 'error',
+                    'react/jsx-uses-vars': 'error',
```

Comment:
I think these rules are already enabled via the `plugin:react/recommended` preset above - so can be removed here?
https://github.com/yannickcr/eslint-plugin-react/blob/a908eb3a1d9804865a7175c31de52637c33d0279/index.js#L127

Reply:
Oddly, if I disable these rules, then I get a bunch of lint errors.  So it seems those rules aren't imported by that plugin for some reason.  Perhaps there's another plugin that does this.  I hope this is ok, but I was thinking I'd just go ahead and merge this and then we can tweak those rules in a follow-up?  I hope that's cool by you.  :)

Reply:
Yeah that's fine :-)